inc/parts/class-content-post_list.php

```diff
@@ -417,6 +421,26 @@ function tc_filter_home_blog_posts_by_tax( $query ) {
          add_filter('tc_grid_expand_featured', '__return_false');
      }
   }
+
+  /**
+  * hook : pre_get_posts
+  * Display home/blog posts in a random order
+  * @return modified query object
+  * @package Customizr
+  * @since Customizr 3.4.18
+  */
+  function tc_display_random_blog_posts( $query ) {
+    // when we have to apply the random order?
+    // in home and blog page
+    if ( ! $query->is_main_query()
+      || ! ( ( is_home() && 'posts' == get_option('show_on_front') ) || $query->is_posts_page ) )
```

Comment:
remove `|| $query->is_posts_page`
if you want to apply it only in the home displaying the blog.